app/exports/common_columns/_provider_information.yml

```diff
@@ -60,3 +66,15 @@ course_code:
 nctl_subject:
   type: array
   maxLength: 255
+
+provider_type:
+  type: string
+  description: The provider type
+  enum:
+    - scitt
+    - lead_school
+    - university
+    - hei
+    - ratified_by_scitt
+    - ratified_by_university
+    - ratified_by_hei
```

Comment:
Same here.

Reply:
So this is the list from the TTAPI

```
  enum provider_type: {
    scitt: "B",
    lead_school: "Y",
    university: "O",
    unknown: "",
    invalid_value: "0", # there is only one of these in the data
  }
```
I think ratified_by_univeristy and ratified_by_hei are the same thing.

I'm just going to put a PIM in to check the data, but i think the ratified values are to do with the provider_type for the accredited_provider